src/stock-adjustment-mobile/add-products-page/add-product-page.jsx

```diff
@@ -163,7 +170,7 @@ const AddProductsPage = ({}) => {
         return (
             <SelectField
                 name={`${fieldName}.lot`}
-                label="Lot Code"
+                label="Lot Code*"
```

Comment:
Why '*' is added? I cannot see the 'required' field. Can we make these required stars red as in mockups?

Reply:
I will check this 'Lot Code' if this is obligatory or not. I can add 'red stars' as in mockups.

Reply:
I added this change with 'red asterisk'